scripts/release

```diff
@@ -407,6 +407,12 @@ publish_release() {
   fi
 }
 
+publish_podspec() {
+  cd "$rootdir"
+  echo "running a pod trunk push..."
+  pod trunk push MaterialComponents.podspec
```

Comment:
Currently we need to pod trunk push --ignore-warnings due to our podspec file
I'm okay adding the --force, but I would also add a //TODO(1234) Remove --force once our podspec file is free of warnings.

Reply:
I'll work on the fixit issue of removing warnings tomorrow and update this PR accordingly after that